apps/web/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import './Icon.scss';
+
+export enum iconColor {
+  Default = 'black',
+  Orange = '#ff2800',
+}
+
+interface Props {
+  children: React.ReactElement;
+  colorProp?: iconColor;
```

Comment:
I think you end up with a more readable API if you keep the prop name and reference string the same. 

```suggestion
  colorProp?: colorProp;
```